be/src/storage/rowset_column_update_state.h

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "storage/olap_common.h"
+#include "storage/primary_index.h"
+#include "storage/rowset/column_iterator.h"
+#include "storage/tablet_updates.h"
+
+namespace starrocks {
+
+class Tablet;
+class DeltaColumnGroup;
+class FileSystem;
+class Segment;
+class RandomAccessFile;
+class ColumnIterator;
+
+struct RowsetSegmentId {
+    RowsetId rowset_id;
+    uint32_t rowset_seg_id;
+    uint32_t segment_id;
+};
+
+struct ColumnPartialUpdateState {
+    // Maintains the mapping of each row of data in the update segment to source row
+    std::vector<uint64_t> src_rss_rowids;
+    bool inited = false;
+    EditVersion read_version;
```

Comment:
Why this member is called read_version?

Reply:
It means that this state is generated by this version, so call it `read_version`.